src/python/pants/util/strutil.py

```diff
@@ -141,6 +141,13 @@ def strip_v2_chroot_path(v: bytes | str) -> str:
     return re.sub(r"/.*/pants-sandbox-[a-zA-Z0-9]+/", "", v)
 
 
+# NB: As above but doesn't convert to str.
+def strip_v2_chroot_path_bytes(v: bytes) -> bytes:
```

Comment:
ijson wants bytes, this satisfies that to avoid keeping sandbox-paths around in the outputs as that breaks later